articles/energy-data-services/how-to-manage-data-security-and-encryption.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Microsoft Energy Data Services - Data Security and Encryption in Microsoft Energy Data Services #Required; page title is displayed in search results. Include the brand.
```

Comment:
Use the Product name as the titlePrefix or as a suffix, but not both

Reply:
fixed.